daisy/resources.go

```diff
@@ -74,20 +74,18 @@ func (rm *baseResourceMap) cleanup() {
 }
 
 func (rm *baseResourceMap) delete(name string) error {
```

Comment:
We could potentially have a race condition with multiple things trying to delete the same resource. I don't know if we'll ever run into this, but this function is now thread-unsafe.

Reply:
Specifically, two threads can execute line 84 simultaneously.